### DISCUSSION TITLE
Picking up landmines?

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#111] [Ideas]
Is it possible to manipulate landmines into items that we can pick up, honestly this is not even a suggestion im just curious as to how far you can manipulate stuff in unity game hacking.

**Comment by winstxnhdw** [comment]
I saw a guy doing it before. I still have no clue how he did it.

**Reply by yujispin990** [reply]
Oh my goodness! its possible!?!

**Reply by winstxnhdw** [reply]
Maybe, but I truly haven't got a clue.
